packages/core/src/App/Containers/RealAccountSignup/add-or-manage-accounts.jsx

```diff
@@ -94,94 +94,102 @@ class AddOrManageAccounts extends React.Component {
     render() {
         if (this.props.is_loading) return <LoadingModal />;
 
-        return (
-            <ThemedScrollbars is_bypassed={isMobile()} autohide={false}>
-                <Tabs
-                    active_index={this.state.active_index}
-                    className='account-wizard add-or-manage tabs--desktop'
-                    onTabItemClick={this.setActiveTabIndex}
-                    top
-                    header_fit_content={isDesktop()}
-                >
-                    <div label={localize('Cryptocurrencies')}>
-                        <div
-                            className={classNames('add-crypto-currency', {
-                                'account-wizard--disabled': this.no_crypto_available,
-                            })}
-                        >
-                            {this.no_crypto_available && (
-                                <div className='account-wizard--disabled-message'>
-                                    <p>
-                                        {localize(
-                                            'You already have an account for each of the cryptocurrencies available on {{deriv}}.',
-                                            {
-                                                deriv: website_name,
-                                            }
-                                        )}
-                                    </p>
-                                </div>
+        const fiat_section = (
```

Comment:
it's better if we define this function outside of this component

Reply:
Then we need to pass all the props to that function which seems unnecessary.